source/Applications/Advanced/PoseConversions/PoseConversions.cs

```diff
@@ -0,0 +1,325 @@
+﻿/*
+This example shows how to perform Hand-Eye calibration.
+*/
+
+using System;
+using System.IO;
+using YamlDotNet.RepresentationModel;
+
+
+using MathNet.Numerics.LinearAlgebra;
+using MathNet.Numerics.LinearAlgebra.Double;
+
+class Program
+{
+    static void Main()
+    {
+        try
+        {
+            var zivid = new Zivid.NET.Application();
+
+            printHeader("This example shows conversions to/from Transformation Matrix");
+
+            // Read a transformation matrix
+            var transformationMatrixFile = Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData)
+                               + "/Zivid/RobotTransform.yaml";
+            Console.WriteLine("Getting PoseState:");
+            var transformationMatrix = readTransform(transformationMatrixFile);
+            var mathMatrix = zividToMathDotNet(transformationMatrix);
```

Comment:
I would also add this to "getTransformationMatrixFromYAML" (see comment on lines 34 to 27) and rename "mathMatrix" to "transformationMatrix" here in main (then it will be the same as the C++ equivalent sample). Also, it will be more clear what this variable is.